src/mmw/js/src/draw/views.js

```diff
@@ -568,6 +573,11 @@ var ResetDrawView = Marionette.ItemView.extend({
     },
 
     resetDrawingState: function() {
+        App.getLeafletMap().eachLayer((l) => {
+            if (l.id && l.id === 'rwd-origin-point') {
+                App.getLeafletMap().removeLayer(l);
+            }
+        });
```

Comment:
Kind of curious why these points aren't handled like other drawn layers. By updating model state.